tdrs-backend/tdpservice/settings/common.py

```diff
@@ -306,16 +306,18 @@ class Common(Configuration):
     # The number of seconds to wait for socket response from clamav-rest
     AV_SCAN_TIMEOUT = os.getenv('AV_SCAN_TIMEOUT', 30)
 
+    s3_src = "s3-us-gov-west-1.amazonaws.com"
```

Comment:
Are wildcards supported here for part of the URL? All of our S3 services currently happen to be in `us-gov-west-1` but that may not always be the case. If allowed I would suggest:
```suggestion
    s3_src = "s3*.amazonaws.com"
```

Reply:
We can't do wildcards mid-domain. We could do `"*.awazonaws.com"` for example, but not what you suggested. One thing we can do potentially is set this by an ENV variable?